src/actions/channel_categories.ts

```diff
@@ -170,6 +170,10 @@ export function moveChannelToCategory(categoryId: string, channelId: string, new
         const category = getCategory(state, categoryId);
         const currentUserId = getCurrentUserId(state);
 
+        const originalCategories = [{
```

Comment:
This name isn't great since it's easy to confuse with `originalCategory` below. I can't think of a better name for this, so maybe `originalCategory` can be renamed to `sourceCategory`?